src/Mod/Draft/draftguitools/gui_togglemodes.py

```diff
@@ -126,15 +126,15 @@ class ToggleDisplayMode(gui_base.GuiCommandNeedsSelection):
 
     def __init__(self):
         super(ToggleDisplayMode,
-              self).__init__(name=translate("draft","Toggle display mode"))
+              self).__init__(name=translate("draft","Toggle Display Mode"))
 
     def GetResources(self):
         """Set icon, menu and tooltip."""
 
         d = {'Pixmap': 'Draft_SwitchMode',
              'Accel': "Shift+Space",
-             'MenuText': QT_TRANSLATE_NOOP("Draft_ToggleDisplayMode","Toggle normal/wireframe display"),
-             'ToolTip': QT_TRANSLATE_NOOP("Draft_ToggleDisplayMode","Switches the display mode of selected objects from flatlines to wireframe and back.\nThis is helpful to quickly visualize objects that are hidden by other objects.\nThis is intended to be used with closed shapes and solids, and doesn't affect open wires.")}
+             'MenuText': QT_TRANSLATE_NOOP("Draft_ToggleDisplayMode","Toggle Wireframe"),
+             'ToolTip': QT_TRANSLATE_NOOP("Draft_ToggleDisplayMode","Switches the view style of the selected objects from flatlines to wireframe and back")}
```

Comment:
"flatlines to wireframe" -> "Flat Lines to Wireframe" (the Display modes have these names).